micronutrientsupport/micronutrient-support-tool
# chore: update dependencies and actions, ui fixes to intervention item and review

- update package dependencies to latest
- update github actions to latest where available
- interventions without content the continue buttons are now correctly shown as disabled
- intervention item review/remove buttons in correct order now

# to test
- intervention item action buttons at right of card, in remove | review order
- when no data available for intervention review (e.g. malawi/biotin), confirm and continue button is disabled visually
